evoting/lib/chains.go

```diff
@@ -1,22 +1,31 @@
 package lib
 
 import (
-	"github.com/dedis/cothority/skipchain"
 	"github.com/dedis/onet"
-)
 
-var client = skipchain.NewClient()
+	"github.com/dedis/cothority/skipchain"
+)
 
-// New creates a new skipchain for a given roster and stores data in the genesis block.
-func New(roster *onet.Roster, data interface{}) (*skipchain.SkipBlock, error) {
-	return client.CreateGenesis(roster, 1, 1, skipchain.VerificationStandard, data, nil)
+// NewSkipchain creates a new skipchain for a given roster and verification function.
+func NewSkipchain(roster *onet.Roster, verifier []skipchain.VerifierID, data interface{}) (
+	*skipchain.SkipBlock, error) {
+	client := skipchain.NewClient()
+	return client.CreateGenesis(roster, 1, 1, verifier, data, nil)
```

Comment:
We should start using higher values for these numbers. Else we cannot take advantage of the _skip_ part in skipchains ;) Something like `8, 4`, which will create a link every 8th block and up to 4096 blocks in the past.

Have a look at this definition: https://github.com/dedis/cothority/blob/master/skipchain/api.go#L100

Reply:
Fixed.